publish-swagger-docs.sh

```diff
@@ -9,7 +9,7 @@ then
 fi
 
 docker-compose up -d
-sleep 10
+sleep 15
```

Comment:
Did the build fail because the sleep was too short?

Reply:
it does for me locally